grails-app/controllers/org/pih/warehouse/RoleInterceptor.groovy

```diff
@@ -13,7 +13,10 @@ import org.pih.warehouse.core.RoleType
  * */
 class RoleInterceptor {
     def userService
-    def dependsOn = [SecurityInterceptor]
```

Comment:
one of the things I don't like about conveniences like this is that Grails never tells you "I don't look at this magic field anymore." This is a filter parameter from Grails 1; interceptors have a global ordering, set immediately below.

Reply:
Exactly what i was thinking.